packages/storage-postgres/migrations/0064_coherence_upvotes.sql

```
@@ -0,0 +1,22 @@
+	"coherence_id" integer NOT NULL,
+	"person_id" integer NOT NULL,
+	"voting_power" numeric(78, 0) NOT NULL,
+	"max_voting_power" numeric(78, 0) NOT NULL,
+	"token_decimals" smallint DEFAULT 0 NOT NULL,
+	"created_at" timestamp DEFAULT now() NOT NULL,
+	"updated_at" timestamp DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "coherence_upvotes" ADD CONSTRAINT "coherence_upvotes_coherence_id_coherences_id_fk"
+  FOREIGN KEY ("coherence_id") REFERENCES "coherences"("id")
+  ON DELETE cascade ON UPDATE no action;
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)